portal/src/graphql/portal/ForgotPasswordSettings.tsx

```diff
@@ -31,28 +36,39 @@ interface ForgotPasswordSettingsState {
   resetCodeExpirySeconds: number;
 }
 
-function constructStateFromAppConfig(
-  appConfig: PortalAPIAppConfig | null
+function constructStateFromAppConfigAndTemplates(
+  appConfig: PortalAPIAppConfig | null,
+  emailAndSmsTemplates: PortalAPIEmailAndSmsTemplates | null
 ): ForgotPasswordSettingsState {
   const forgot_password = appConfig?.forgot_password;
 
   return {
-    emailHtmlTemplate: "", // TODO: handle email template
-    emailPlainTextTemplate: "", // TODO: handle email template
-    smsTemplate: "", // TODO: handle sms template
+    emailHtmlTemplate: emailAndSmsTemplates?.emailHtml ?? "",
+    emailPlainTextTemplate: emailAndSmsTemplates?.emailHtml ?? "",
```

Comment:
`emailHtmlTemplate`, `emailPlainTextTemplate` are same, both equals `emailAndSmsTemplates?.emailHtml`?

Reply:
should be `emailAndSmsTemplates?.emailText` ?